src/common/form/FormStore.js

```diff
@@ -12,8 +12,9 @@ type FormStoreProps = {|
 |};
 
 type InitFieldProps = {|
-  name: string,
   initialValue?: any,
```

Comment:
Isn't this an object? This could be typed better than `any`

Reply:
Yeah..I'll make generics for form/FormStore/FormField etc in the future.